README.md

```diff
@@ -94,6 +94,15 @@ For Windows, you can use the following commands:
 * `gradlew spotlessApply` an check and apply formatting to any plain-text file.</br>
 * `gradlew build`  provides a command line to execute build script.</br>
 
+To get the latest apk fom the Github actions artifacts, follow these steps
```

Comment:
Make this an eye catching title via making it some h3/h2/bold with lesser than 7 words, as it is very difficult to find this procedure in the readme for downloading

Reply:
> Make this an eye catching title via making it some h3/h2/bold with lesser than 7 words, as it is very difficult to find this procedure in the readme for downloading

@s-ayush2903 Hpw about this ?